src/Server/OutputBuilder.cs

```diff
@@ -23,13 +24,37 @@ public class OutputBuilder
 
         private int bufferPos;
         private int charsCapacity;
+        
+        //Static Char Numbers
+        private static readonly char[] CharNumbers = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'};
+
+        private static readonly string[] CharBool = {"False", "True"};
```

Comment:
Why represent as an array?  I think this just ends up producing extra array indexing at runtime, and doesn't really make it any clearer IMO.